clang/include/clang/Basic/LangStandard.h

```diff
@@ -130,6 +130,12 @@ struct LangStandard {
   /// hasDigraphs - Language supports digraphs.
   bool hasDigraphs() const { return Flags & Digraphs; }
 
+  /// hasRawStringLiterals - Language supports R"()" raw string literals.
+  bool hasRawStringLiterals() const {
+    // GCC supports raw string literals in C, but not in C++ before C++11.
+    return isCPlusPlus11() || (!isCPlusPlus() && isGNUMode());
```

Comment:
Update: This seems to be intended, so I suppose we should only support them in `-gnu99` mode and later.

Reply:
okay, so my guess was correct. It's still odd though because that same logic applies to C99 (it's "old" and the behavior can still change). But yeah, we should follow the same behavior as GCC in this case (less confusion for users).